neptune/sessions.py

```diff
@@ -64,6 +65,15 @@ def get_project(self, project_qualified_name):
             name=project.name
         )
 
+    def get_default_project(self, organization_name=None):
```

Comment:
Why do we need this in `Session`'s API? I'd drop it.

Reply:
`get_default_project` is called when user initializes session without `project_qualified_name` argument. I can drop it but it that case we'll need to find another solution to support flow like:
```python
import neptune

neptune.init()
neptune.create_experiment(...)
```

Reply:
I think it's better to inline it in `init()`. We don't want to expose such functionality, it's here only to facilitate the simplest-possible API.